src/graph_visualization/graph_neural_network.zig

```diff
@@ -72,7 +79,7 @@ pub fn graphNeuralNetwork(
         if (label_index > color_pair_map.len - 1) {
             return error.ColorPairMapNotLargeEnough;
         }
-        const pixel_color: u24 = color_pair_map[label_index].secondary_color;
+        const pixel_color: u24 = color_pair_map[label_index].primary_color;
```

Comment:
Fixed the graph rendering the training and test points as the same color

This regressed in https://github.com/MadLittleMods/zig-ocr-neural-network/pull/27 when we tried to make things generic